src/Workspaces/MSBuildTest/VisualStudioMSBuildWorkspaceTests.cs

```diff
@@ -254,7 +254,7 @@ public async Task TestChecksumAlgorithm_NonDefault(string language)
         {
             var files = language == LanguageNames.CSharp ?
                 GetSimpleCSharpSolutionFiles().WithFile(@"CSharpProject\CSharpProject.csproj", Resources.ProjectFiles.CSharp.WithChecksumAlgorithm) :
-                GetSimpleVisualBasicSolutionFiles().WithFile(@"VisualBasicProject\VisualBasicProject.csproj", Resources.ProjectFiles.VisualBasic.WithChecksumAlgorithm);
+                GetSimpleVisualBasicSolutionFiles().WithFile(@"VisualBasicProject\VisualBasicProject.vbproj", Resources.ProjectFiles.VisualBasic.WithChecksumAlgorithm);
```

Comment:
Whoopsie .... I'm scared to `git blame` this line cause 99% sure it's gonna be my fault.

Reply:
Nope, @tmat recently added support for this and also added tests. But since the tests never ran in CI...they didn't actually work.